docs/resources/Audit_Log.md

```diff
@@ -132,12 +132,21 @@ Whenever an admin action is performed on the API, an entry is added to the respe
 | temporary                     | [invite](#DOCS_RESOURCES_INVITE/invite-metadata-object) | boolean                                                                                  | invite code is temporary/never expires                                                                                                                  |
 | deaf                          | [user](#DOCS_RESOURCES_USER/user-object)                | boolean                                                                                  | user server deafened/undeafened                                                                                                                         |
-| mute                          | [user](#DOCS_RESOURCES_USER/user-object)                | boolean                                                                                  | user server muted/unmuted                                                                                                                              |
+| mute                          | [user](#DOCS_RESOURCES_USER/user-object)                | boolean                                                                                  | user server muted/unmuted                                                                                                                               |
 | nick                          | [user](#DOCS_RESOURCES_USER/user-object)                | string                                                                                   | user nickname changed                                                                                                                                   |
 | avatar_hash                   | [user](#DOCS_RESOURCES_USER/user-object)                | string                                                                                   | user avatar changed                                                                                                                                     |
 | id                            | any                                                     | snowflake                                                                                | the id of the changed entity - sometimes used in conjunction with other keys                                                                            |
 | type                          | any                                                     | integer ([channel type](#DOCS_RESOURCES_CHANNEL/channel-object-channel-types)) or string | type of entity created                                                                                                                                  |
 
+###### Example Partial Channel Object
```

Comment:
Shouldn't this be "Partial Role Object' if this is to clarify that Role objects can be partial?